datasets/cord-19.yaml

```diff
@@ -0,0 +1,23 @@
+Name: COVID-19 Open Research Dataset (CORD-19)
+Description: Full-text and metadata dataset of COVID-19 and coronavirus-related research articles optimized for machine readability.
+Documentation: https://pages.semanticscholar.org/coronavirus-research
+Contact: partnerships@allenai.org
+ManagedBy: Allen Institute for AI
+UpdateFrequency: Weekly
+Tags:
+  - COVID-19
+  - coronavirus
+  - SARS
+  - MERS
+License: Open (see license file for details)
+Resources:
+  - Description: S3 bucket with CORD-19 dataset files
+    ARN: ai2-semanticscholar-cord-19
+    Region: us-west-2
+    Type: S3
+DataAtWork:
+  Tools & Applications:
+    - Title: COVID-19 Open Research Dataset Challenge (CORD-19)
+      URL: https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge
+      AuthorName: Kaggle
+      AuthorURL: N/A
```

Comment:
AuthorURL is optional, just do not include if no relevant URL

Reply:
Done!